src/components/Table/GridTable.tsx

```diff
@@ -296,37 +293,36 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   const columnSizes = useSetupColumnSizes(style, columns, tableRef, resizeTarget);
 
   // Filter + flatten + component-ize the sorted rows.
-  let [headerRows, filteredRows]: [RowTuple<R>[], RowTuple<R>[]] = useMemo(() => {
+  let [headerRows, filteredRows]: [RowTuple<R>[], RowTuple<R>[]] = useComputed(() => {
```

Comment:
Changed this to a `useComputed` so that we re-calc "which rows are we showing?" each time the `collapsedRows` observable changes.